Jordan Espinal

```diff
@@ -0,0 +1,3 @@
+Python was my first
+CSS was next
+Now is Java Script
```

Comment:
Remove the space between Java and Script please.

Reply:
Will do!